ui/views_test.py

```diff
@@ -91,13 +91,14 @@ def test_authenticated_user_doesnt_redirect(self):
     def test_webpack_url(self):
```

Comment:
I know you did not write this test, but (as note for the future), it is better to use `ddt` or `@pytest.mark.parametrize` for multiple inputs for a test.